templates/operator-day/kubecon-na-2021.html

```diff
@@ -50,11 +50,11 @@ <h3>October 12th</h3>
           </tr>
           <tr>
             <td>3:45 PM - 4:00 PM</td>
-            <td><b>Removing Toil: Improvements in the Charmed Operator ecosystem</b><br/>Pete Vander Giessen</td>
+            <td><b>Removing Toil: Improvements in the charm ecosystem</b><br/>Pete Vander Giessen</td>
           </tr>
           <tr>
             <td></td>
-            <td colspan="2">The Charmed Operator Framework is undergoing constant development. What are the key goals of the charmed operator framework? What's happened since the last Operator Day? What's coming up in the next few months? Join this session to learn more about Juju, Pebble, Charmcraft, Charmed Operators and more.</td>
+            <td colspan="2">Juju is undergoing constant development. What are the key goals of the <code>ops</code>> library? What's happened since the last Operator Day? What's coming up in the next few months? Join this session to learn more about Juju, Pebble, Charmcraft, Charmed Operators and more.</td>
```

Comment:
```suggestion
            <td colspan="2">Juju is undergoing constant development. What are the key goals of the <code>ops</code> library? What's happened since the last Operator Day? What's coming up in the next few months? Join this session to learn more about Juju, Pebble, Charmcraft, Charmed Operators and more.</td>
```

Extra `>`